### PR TITLE
i18n(pt-br): translating guides/pages file

### PR DESCRIPTION
**Pull request by kayaman**
#### Description

- Adding the translation of the `guides/pages.mdx` missing file.

### CODE REVIEW

**Review by Fryuni — CHANGES_REQUESTED** [review]
Thank you for the PR!

Some things to improve:
- There is a missing sentence and an English sentence that was kept around.
- Internal links should always point to the translated pages in the same language, even if the target of the link hasn't been translated yet. You can check those locally by running `CHECK_LINKS=true pnpm build`.
- We translate some terms differently in other pages, so I sent suggestions for consistency
- Some terms that we usually translate elsewhere were not translated

**Comment by kayaman** [comment]
@Fryuni thank you for the review. I did the requested changes but there's still one issue I wasn't able to solve, as pointed here: https://github.com/withastro/starlight/actions/runs/8299714503/job/22716051747#step:6:484
Do you have a clue, please? Thank you.

**Comment by kayaman** [comment]
> @Fryuni thank you for the review. I did the requested changes but there's still one issue I wasn't able to solve, as pointed here: https://github.com/withastro/starlight/actions/runs/8299714503/job/22716051747#step:6:484 Do you have a clue, please? Thank you.

I did dig a little more and ended with just one last broken link.

File: docs/src/content/docs/pt-br/guides/pages.mdx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+**padrão:** a linguagem da localidade atual
+
+Define a tag de linguagem BCP-47 para o conteúdo da página, por exemplo, `en`, `zh-CN`, ou `pt-BR`.
+
+##### `isFallback`
+
+**tipo:** `boolean`  
+**padrão:** `false`
+
+Indica se esta página está usando [conteúdo fallback](/pt-br/guides/i18n/#fallback-content) porque não há tradução para a linguagem atual.
```
Comment (HiDeoo) [inline docs/src/content/docs/pt-br/guides/pages.mdx:174]:
```suggestion
Indica se esta página está usando [conteúdo fallback](/pt-br/guides/i18n/#conteúdo-de-fallback) porque não há tradução para a linguagem atual.
```

> I did dig a little more and ended with just one last broken link.

Thanks for the updates 🙌 

It looks like the last broken link is this one as in `/pt-br/guides/i18n/#fallback-content`, the `fallback-content` does not exist but is instead [`conteúdo-de-fallback`](https://starlight.astro.build/pt-br/guides/i18n/#conte%C3%BAdo-de-fallback) as it's based on the section title which is translated too.

**Comment by kayaman** [comment]
> > @Fryuni thank you for the review. I did the requested changes but there's still one issue I wasn't able to solve, as pointed here: https://github.com/withastro/starlight/actions/runs/8299714503/job/22716051747#step:6:484 Do you have a clue, please? Thank you.
> 
> 
> 
> I did dig a little more and ended with just one last broken link.

awesome @Fryuni and @HiDeoo ! Thank you for your time reviewing it. I think all tests are fixes now. Thank you so much.

**Review by HiDeoo — APPROVED** [review]
Thanks for fixing the remaining issues. This looks perfect now :thumbsup:

Great job on the translation and the reviews everyone :rocket:

**Review by Fryuni — APPROVED** [review]
Amazing, looking great now!

**Pull request merged**